page.go

```diff
@@ -89,10 +88,10 @@ type CategoryPage struct {
 
 // getPage GETs the data at the specified URL and unmarshals it into page.
 func (c *Client) getPage(url string, page interface{}) error {
```

Comment:
Can we remove `getPage` now since it's functionally equivalent to the `get` method you've added?